applications/filestore-backup/Chart.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: filestore-backup
+version: 1.0.0
+description: Tool to manage Google Filestore backups
```

While this is a description of the underlying repository, I think it's not quite a description of the Kubernetes service being defined here. It's a `CronJob` so it's not really a tool. Maybe something like:

```suggestion
description: Create and manage lifecycle of Google Filestore backups
```